src/toolkits/drawing_classifier/drawing_classifier.hpp

```diff
@@ -170,19 +186,80 @@ class EXPORT drawing_classifier: public ml_model_base {
   REGISTER_CLASS_MEMBER_FUNCTION(drawing_classifier::export_to_coreml,
                                  "filename");
 
+  REGISTER_CLASS_MEMBER_FUNCTION(drawing_classifier::init_train, "data",
+                                  "target_column_name", "feature_column_name",
+                                  "validation_data", "options");
+  register_defaults("init_train",
```

Comment:
init train is not exposed to python frontend, isn't it?

Reply:
Following the OD workflow, `init_train` is exposed to the Python frontend (for eventual use in checkpointing). I considered we'd want to follow that model.

Reply:
well, why people need a checkpoint at init_train ??? Init_train is cheap to call.

Reply:
Done.